typings/index.d.ts

```diff
@@ -0,0 +1,144 @@
+        DJSlib?: "v12" | "v11";
+        default?: GiveawayStartOptions;
+        botsCanWin?: boolean;
+        exemptPermissions?: PermissionResolvable[];
+        exemptMembers?: () => boolean;
+        embedColor?: ColorResolvable;
+        embedColorEnd?: ColorResolvable;
+        reaction?: string;
+    }
+    interface GiveawayStartOptions {
+        time: number;
+        winnerCount: number;
+        prize: string;
+        hostedBy?: User;
+        botsCanWin?: boolean;
+        exemptPermissions?: PermissionResolvable[];
+        exemptMembers?: () => boolean;
+        embedColor?: ColorResolvable;
+        embedColorEnd?: ColorResolvable;
+        reaction?: string;
+        messages?: GiveawaysMessages;
+    }
+    interface GiveawaysMessages {
+        giveaway?: string;
+        giveawayEnded?: string;
+        inviteToParticipate?: string;
+        timeRemaining?: string;
+        winMessage?: string;
+        embedFooter?: string;
+        noWinner?: string;
+        winners?: string;
+        endedAt?: string;
+        hostedBy?: string;
+        units?: {
+            seconds?: string;
+            minutes?: string;
+            hours?: string;
+            days?: string;
+            pluralS?: false;
+        };
+    }
+    interface GiveawaysManagerEvents {
+        giveawayEnded: [Giveaway, GuildMember[]];
+        giveawayReactionAdded: [Giveaway, GuildMember, MessageReaction];
+        giveawayReactionRemoved: [Giveaway, GuildMember, MessageReaction];
+    }
+    class Giveaway extends EventEmitter {
+        constructor(manager: GiveawaysManager, options: GiveawayData)
+
+        public botsCanWin: boolean;
+        readonly channel: TextChannel;
+        public channelID: Snowflake;
+        public client: Client;
+        readonly content: string;
+        public data: GiveawayData;
+        public embedColor: ColorResolvable;
+        public embedColorEnd: ColorResolvable;
+        public endAt: number;
+        public ended: boolean;
+        public exemptPermissions: PermissionResolvable[];
+        readonly giveawayDuration: number;
+        public guildID: Snowflake;
+        public hostedBy: string | null;
+        public manager: GiveawaysManager;
+        public message: Message | null;
+        public messageID: Snowflake | null;
+        public messages: GiveawaysMessages;
+        public options: GiveawayData;
+        public prize: string;
+        readonly remainingTime: number;
+        public startAt: number;
+        public winnerCount: number;
+
+        public exemptMembers(): boolean;
+        public edit(options: GiveawayEditOptions): Promise<Giveaway>;
+        public end(): Promise<GuildMember[]>
+        // @ts-ignore-next-line
+        public async fetchMessages(): Promise<Message>
+        public reroll(options: GiveawayRerollOptions): Promise<GuildMember[]>
+        // @ts-ignore-next-line
+        public async roll(winnerCount?: number): Promise<GuildMember | GuildMember[]>;
```

Comment:
I don't see a case where a single member could be returned
```suggestion
        public async roll(winnerCount?: number): Promise<GuildMember[]>;
```